service/history/hsm/tree.go

```diff
@@ -77,6 +79,11 @@ type cachedMachine struct {
 	outputs []TransitionOutput
 }
 
+type NodeBackend interface {
```

Comment:
Should probably add a description to match the format of the other structs here.

Reply:
I missed this. Thanks.